Lib/asyncio/proactor_events.py

```diff
@@ -858,6 +858,10 @@ def loop(f=None):
                 if self.is_closed():
                     return
                 f = self._proactor.accept(sock)
+            except exceptions.CancelledError:
+                # Effectively ignore connections that throw a cancelled error
+                # during setup, loop back around and continue serving.
+                sock.close()
```

Comment:
Suppressing `CancelledError` is a huge red flag and should only be done in desperate times. Can you write an elaborate comment what's going on here? I don't understand why `CancelledError` needs suppressing neither from this comment nor from the PR description.

Reply:
It looks like this is supposed to be:
```python
except ConnectionResetError:
    self.call_soon(loop)
```

https://github.com/python/cpython/pull/124032/files#diff-15176d64608861034db7168743f4f116f4cb33422cf685a2b911481c55826aa2R861-R862

We have reproduced this: the regular uvicorn+fastapi server receives a connection from a process that is forcefully terminated afterward.
What `_loop` needs to do: determine if an error = `OSError: [WinError 64] The specified network name is no longer available`; in that case, it should just run the next iteration of the loop.

Our working fix looks like that: https://gist.github.com/cybergrind/9cbbdc94503548d74dc4d5d3ed99248c